README.md

```diff
@@ -6,6 +6,12 @@ Website of the Czech PyLadies chapter / Web českých PyLadies.
 
 *Pokud je kontrolka zelená, povedlo se dostat poslední změny z větve `master` do ostrého provozu. Má-li jinou barvu, stala se [někde po cestě](https://travis-ci.org/PyLadiesCZ/pyladies.cz) chyba.*
 
+
+## License
```

Comment:
hmmm je to komentar nebo fakt navrh na zmenu? Na bitbucket jsou tyhle veci prehlednejsi, musim uznat.